cmd/rpc.go

```diff
@@ -0,0 +1,58 @@
+package cmd
```

Comment:
[discussion]
Another idea is to decouple `cmd/binary` from cli/cmd utils. Basically, move all the `cmd/...go` files to `cli` or `lib/cli`, then `cmd` would only be a place for our main apps/entrypoints. 
Alternatively, we can move them to `nodebuilder/node/cmd`. I just looked at we have there and all the stuff there are node/admin related cmds - start, auth, utils for them, etc. So it logically makes sense to keep it there.
Generally, I like the idea of not keeping the `cmd` pkg with 0 code and only subpkgs with binaries, which is also idiomatic.

Reply:
Agree, but let's do it in a subsequent PR.

Reply:
https://github.com/celestiaorg/celestia-node/issues/2754